src/ConfigParser/ConfigParser.js

```diff
@@ -643,4 +542,76 @@ function featureToPlugin (featureElement) {
 
     return plugin;
 }
+
+class BaseResource {
+    constructor (attrs, { platform = null } = {}) {
+        // null means resource is shared between platforms
+        this.platform = platform || null;
+        this.src = attrs.src;
+        this.target = attrs.target || undefined;
+    }
+}
+
+class ImageResource extends BaseResource {
+    constructor (attrs, opts) {
+        super(attrs, opts);
+        this.density = attrs.density;
+        this.width = Number(attrs.width) || undefined;
+        this.height = Number(attrs.height) || undefined;
+        this.background = attrs.background || undefined;
+        this.foreground = attrs.foreground || undefined;
+    }
```

Comment:
If you want to add a doc block for in-code for documentation purpose, you can mention that the `density` , `background`, and `foreground` attributes/properties are only for the Android platform.